config/configmap/inferenceservice.yaml

```diff
@@ -500,3 +500,11 @@ data:
       "enableMetricAggregation": "false",
       "enablePrometheusScraping" : "false"
     }
+  
+  # Add http_proxy/HTTP_PROXY, https_proxy/HTTP_PROXY, no_proxy/NO_PROXY values as per your organization
+  http_proxy: "http://your-web-proxy-url:port"
+  https_proxy: "https://your-web-proxy-url:port"
+  no_proxy: ".your-internal-domain"
+  HTTP_PROXY: "http://your-web-proxy-url:port"
+  HTTPS_PROXY: "https://your-web-proxy-url:port"
+  NO_PROXY: ".your-internal-domain"
```

Comment:
I think this would not work, because the KServe controller does not know how to inject these configurations to the inference service pod. From the example you showed in the description seems like you are creating the inference service pod directly without using the inference service custom resource which is not normally how KServe is used.